tests/Test/Utilities.hs

```diff
@@ -35,7 +34,7 @@ testSequencing =
                 ]
     in
         testCase "Test lsequence"
-        $ (List.head .> Tuple.fst) <$> result >>= assertEq "a"
+        $ (List.head .> fst) <$> result >>= assertEq "a"
```

Comment:
could actually replace this with `head` since it is also in the default imports (`Prelude`)